CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+  project once.
+- **respect** your abilities, interests, and personal differences.  Our
+  different talents documenting, testing, coding, reviewing, reporting, and/or
+  evangelizing are all valuable to the project.
+- **make concise and actionable points** to respect your time as a volunteer.
+  We respect each other's time by keeping the signal-to-noise ratio high,
+  staying on-topic, and being a good house-guest: every visit we strive to
+  leave the place in better shape than when we arrived.
+- **are kind** and **assume positive intent**, even when we disagree while
+  working towards consensus and achieving the project's technical goals.
+- **educate and illuminate** others with something we know more about. We were
+  all beginners once, and can teach and learn through feedback and sharing.
+- **are open to feedback** from others, be it technical points, interpersonal
+  tips, or language/wording suggestions.
+- **accept responsibility** by apologizing to those affected by our mistakes
+  and graciously accept apologies knowing that mistakes are part of the
+  process.
+- **use words carefully**, because we may not understand irony or negative
+  sarcasm. We communicate plainly, and we justify our positions objectively and
+  explicitly.
+- **leave with class**, without drama or bitterness, when resigning for any
+  reason. We champion your rights under the terms of the GPL to fork the code
+  and create a new project, just as we stand on the shoulders of those who gave
+  us the right to create this project.
+
+## Reporting Issues
+
+Please bring up issues by contacting
+[dreamer](mailto:patryk.obara+coc@gmail.com) or
+[krcroft](mailto:k.r.croft+coc@gmail.com) who will handle your report with
+discretion. Your report should include:
+
+- Names (real or usernames) of those involved in the incident.
+- Please provide links or transcripts of the publicly available records (e.g. a
+  mailing list archive, GitHub issue, public IRC log, or Discord transcript),
+  so that we can review the evidence. Private communication is outside our
+  scope, we suggest utilizing the blocking features on the platform in
+  question.
+- Any additional information that may be helpful to achieve resolution.
+
+We will attempt to resolve the issue publically by refering to the CoC and
+suggesting corrective actions. We will also try to reduce the distraction and
+impact to those affected.
+
+### Scope
+
+This goal of this document is to preserve the well being of the project and its
+participants and we will restrict or remove those who disrupt them.
+
+This code does not replace the terms of service or acceptable use policies that
+are provided by the websites used to support this community. Nor does this code
+apply to communications or actions that take place outside of the context of
+this community. This code is a social-contract that informs participants of our
+social expectations. It is not a terms of service or legal contract.
+
```

Comment:
This paragraph does not define "Scope" really - maybe if we had some official social media presence, separate services for handling documentation or wiki then scope of the document would need to be described - so maybe sometime in future it will be required

In the spirit of "less is more", I suggest removing the section - our goals are already explained in the very beginning :)